DMCompiler/DMStandard/Types/Matrix.dm

```diff
@@ -11,22 +11,7 @@
 	proc/Interpolate(Matrix2, t)
 		set opendream_unimplemented = TRUE
 
-	New(var/a = 1, var/b = 0, var/c = 0, var/d = 0, var/e = 1, var/f = 0)
-		if (istype(a, /matrix))
-			var/matrix/mat = a
-			src.a = mat.a
-			src.b = mat.b
-			src.c = mat.c
-			src.d = mat.d
-			src.e = mat.e
-			src.f = mat.f
-		else
-			src.a = a
-			src.b = b
-			src.c = c
-			src.d = d
-			src.e = e
-			src.f = f
+	New(var/a, var/b, var/c, var/d, var/e, var/f)
```

Comment:
These default values weren't actually a thing, and you can't use named arguments with `matrix()` anyways